ConsoleApplication2.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup />
+</Project>
```

Comment:
you should never upload such files to the repository (it's even mentioned in each home task description). This is your own environment files that makes no sense for example for users that are not using visual studio.
You can configure .gitignore file to avoid it automatically